src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -36,10 +34,13 @@ object WorkflowActor {
 case class WorkflowActor(id: WorkflowId,
                          namespace: WdlNamespace,
                          actualInputs: WorkflowCoercedInputs,
-                         backend: Backend) extends LoggingFSM[WorkflowState, WorkflowFailure] {
+                         backend: Backend) extends LoggingFSM[WorkflowState, WorkflowFailure] with WorkflowDescriptor {
+
   implicit val timeout = Timeout(5 seconds)
 
-  private val storeActor = context.actorOf(StoreActor.props(namespace, actualInputs))
+  private val storeActor = context.actorOf(StoreActor.props(namespace, backend.initializeWorkflow(this, actualInputs)))
```

Comment:
Technically yes, but only through the `WorkflowDescriptor` interface.  So a malicious receiver would need to downcast it to an `Actor` in order to do bad things.  I could pass in an anonymous instantiation of the `WorkflowDescriptor` trait if you think there's an issue with this.